metagraph/src/graph/representation/succinct/boss.hpp

```diff
@@ -283,10 +283,10 @@ class BOSS {
      * Representation states of the BOSS table.
      *
      * SMALL is the smallest
-     * STAT provides a good space/time tradeoff
+     * STAT provides the best space/time trade-off
      * FAST is the fastest but large
```

Comment:
For better clarity, maybe it would be useful to also describe which encoding is used for each of L and W in these states

Reply:
same in `dbg_succinct.hpp`

Reply:
done